nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-client-dto/src/main/java/org/apache/nifi/web/api/dto/CounterDTO.java

```diff
@@ -98,4 +98,12 @@ public void setValueCount(Long valueCount) {
         this.valueCount = valueCount;
     }
 
+    @Override
+    public CounterDTO clone() {
```

Comment:
Was this the intended implementation of this method? This implementation doesn't seem consistent with others.